exchanges/lbank/lbank_wrapper.go

```diff
@@ -198,62 +197,42 @@ func (l *Lbank) UpdateTicker(ctx context.Context, p currency.Pair, a asset.Item)
 
 // UpdateOrderbook updates and returns the orderbook for a currency pair
 func (l *Lbank) UpdateOrderbook(ctx context.Context, p currency.Pair, assetType asset.Item) (*orderbook.Base, error) {
-	if p.IsEmpty() {
-		return nil, currency.ErrCurrencyPairEmpty
+	if !l.SupportsAsset(assetType) {
+		return nil, fmt.Errorf("%w: %q", asset.ErrNotSupported, assetType)
 	}
-	if err := l.CurrencyPairs.IsAssetEnabled(assetType); err != nil {
+
+	fPair, err := l.FormatExchangeCurrency(p, assetType)
```

Comment:
FormatExchangeCurrency this already does the check of l.SupportsAsset() call and can remove the l.SupportsAsset() call above.

Reply:
I tried this with asset.Options and looks like we don't hit this (or any error) with it removed, so decided to leave it in

Reply:
That should be an error as long as we are using the exchange `l.Supp` to format, right?

Reply:
May be we can log this as an issue.